v2/client/reservation.go

```diff
@@ -95,6 +96,23 @@ func Reserve(ctx context.Context, h host.Host, ai peer.AddrInfo) (*Reservation,
 		result.Addrs = append(result.Addrs, a)
 	}
 
+	voucherBytes := rsvp.GetVoucher()
```

Comment:
@vyzo We also need code in the v2 dialer that validates a voucher before dialing.

Reply:
not yet, it's *very* premature -- first we have to figure out how to distribute the vouchers.